.github/workflows/deploy-dojo-for-e2e.yml

```
@@ -0,0 +1,316 @@
+        run: |
+          aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 058264219587.dkr.ecr.us-east-1.amazonaws.com
+
+      - name: (Dojo) Setup pnpm
+        if: matrix.app.type == 'dojo'
+        uses: pnpm/action-setup@v4
+        with:
+          version: "9.5"
+
+      - name: (Dojo) Use Node.js 22
+        if: matrix.app.type == 'dojo'
+        uses: actions/setup-node@v2
```

actionlint (1.7.7)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)